### PR TITLE
Omicsarea tree in homepage

### PR DESCRIPTION
**Pull request by thomasdenecker**
- [x] Add tree chart in homepage

### CODE REVIEW

File: apps/core/templates/core/home-authenticated.html [hunk 1]
```diff
@@ -29,6 +32,17 @@ <h2>{% trans "Welcome to Pixel" %}</h2>
         ),
         { title: '{{ pixels_by_species.title }}' }
       );
+
+      var omicsareaTree = new google.visualization.OrgChart(
+        document.getElementById('omicsarea-tree')
+      );
+      omicsareaTree.draw(
+        google.visualization.arrayToDataTable(
+          {{ Omicsarea_tree.data|safe }},
```
Comment (willdurand) [inline apps/core/templates/core/home-authenticated.html]: `Omicsarea_tree` => `omics_area_tree`

File: apps/core/templates/core/home-authenticated.html [hunk 2]
```diff
@@ -29,6 +32,17 @@ <h2>{% trans "Welcome to Pixel" %}</h2>
...
+          true
+        ),
+        { title: '{{ Omicsarea_tree.title }}' }
```
Comment (willdurand) [inline apps/core/templates/core/home-authenticated.html]: `Omicsarea_tree` => `omics_area_tree`

File: apps/core/views.py [hunk 3]
```diff
@@ -26,11 +26,24 @@ def get(self, request, *args, **kwargs):
                 ' GROUP BY core_species.name'
             ))
 
+            Omicsarea_tree = Pixel.objects.raw((
```
Comment (willdurand) [inline apps/core/views.py]: `Omicsarea_tree` => `omics_area_tree`

File: apps/core/views.py [hunk 4]
```diff
@@ -26,11 +26,24 @@ def get(self, request, *args, **kwargs):
...
+                "Select A.name as area, COALESCE(B.name, 'Area') as parent, "
+                " 1 as id"  # this is a hack to overcome Django ORM limitations
+                " from core_omicsarea A"
```
Comment (willdurand) [inline apps/core/views.py]: I dont think you should use `A` or `B`, better using the table names.

File: apps/core/views.py [hunk 5]
```diff
@@ -26,11 +26,24 @@ def get(self, request, *args, **kwargs):
...
+                " ON a.parent_id = B.id;"
+            ))
+
             return render(request, 'core/home-authenticated.html', {
                 'pixels_by_species': {
                     'title': _('Pixels by Species'),
                     'data': [[row.name, row.nb] for row in pixels_by_species],
                 },
+
+                'Omicsarea_tree': {
```
Comment (willdurand) [inline apps/core/views.py]: `Omicsarea_tree` => `omics_area_tree`

File: assets/scss/_home.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#omicsarea-tree tr{
```
Comment (willdurand) [inline assets/scss/_home.scss]: missing space before `{`

File: assets/scss/_home.scss [hunk 7]
```diff
@@ -0,0 +1,3 @@
...
+  background: white;
```
Comment (willdurand) [inline assets/scss/_home.scss]: Why is this necessary actually?

Comment (thomasdenecker) [reply assets/scss/_home.scss]: The tree creation function creates an HTML table. We use foundation for the style and it automatically applies its style in all tables and therefore the tree. Without this CSS, we'd have gray bars in the background.

File: apps/core/views.py [hunk 8]
```diff
@@ -26,11 +26,26 @@ def get(self, request, *args, **kwargs):
                 ' GROUP BY core_species.name'
             ))
 
+            omics_area_tree = Pixel.objects.raw((
+                "Select core_omicsarea_1.name as area,"
```
Comment (willdurand) [inline apps/core/views.py]: SELECT, FORM and AS could be uppercased.

File: apps/core/views.py [hunk 9]
```diff
@@ -26,11 +26,26 @@ def get(self, request, *args, **kwargs):
...
+                " COALESCE(core_omicsarea_2.name, 'Area') as parent, "
+                " 1 as id"  # this is a hack to overcome Django ORM limitations
+                " from core_omicsarea core_omicsarea_1"
```
Comment (willdurand) [inline apps/core/views.py]: no need to use an alias  here, simply use the table name as is.

Example:

```
SELECT table.column
FROM table
WHERE table.other_column = 2
```
